gradio/chat_interface.py

```diff
@@ -202,7 +202,10 @@ def __init__(
             with Group():
                 with Row():
                     if textbox:
-                        textbox.container = False
+                        if self.multimodal:
```

Comment:
This makes the textbox look squished in the multimodal case

<img width="1212" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/a2ca71d4-c969-47fe-afe8-e37b6f24aaad">

I think we just need `submit_btn=None` in this case

Reply:
That's strange, I flipped the lines. Good catch. Should be good now.